core/src/luigi-config/feature-discovery.js

```diff
@@ -115,7 +115,7 @@ export async function updateFeature(featureName) {
       const activeCluster = getActiveCluster();
       const permissionSet = await fetchPermissions(
         getAuthData(),
-        getCurrentContextNamespace(activeCluster.kubeconfig),
+        getCurrentContextNamespace(activeCluster?.kubeconfig),
```

Comment:
that function is trying to get current-context from this argument, it will lead to another error
```
const currentContextName = kubeconfig['current-context'];
```